tests/test_tasks/test_supervised_task.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+import openml
+from tests.test_tasks import OpenMLTaskTest
+
+
+class OpenMLSupervisedTaskTest(OpenMLTaskTest):
+    # task id will be set from the
+    # extending classes
+
+    def setUp(self):
+
+        super(OpenMLSupervisedTaskTest, self).setUp()
+        self.task_id = 11
+
+    @classmethod
+    def setUpClass(cls):
+        if cls is OpenMLSupervisedTaskTest:
+            raise unittest.SkipTest(
+                "Skip OpenMLSupervisedTaskTest tests,"
+                " it's a base class"
+            )
+        super(OpenMLSupervisedTaskTest, cls).setUpClass()
+
+    def test_get_X_and_Y(self):
```

Comment:
Could you please rename this method? I find the name misleading.

Reply:
I can, I did not change the naming since that was the previous name the method had.
What do you think about **test_get_examples_and_labels**?

Reply:
That would be a descriptive name.

Reply:
@mfeurer Now that I am thinking about it, we are testing the class method and the method that we are testing is **get_X_and_y()**.
Should we still rename?

Reply:
Could you please provide more context? I can't follow you right now.